src/api/backend.js

```diff
@@ -1,14 +1,16 @@
 import axios from 'axios'
 
 const API_BASE_URL = setBaseURLWithDefaultOrEnvValue()
+const API_VERSION = 'v1.0.0'
 
 function canReadURLFromEnv () {
   return !!process.env['API_BASE_URL']
 }
 
 export function setBaseURLWithDefaultOrEnvValue () {
   const defaultUrl = 'https://kubeportal-dev.api.datexis.com/'
-  return canReadURLFromEnv() ? process.env['VUE_APP_BASE_URL'] : defaultUrl
+  const baseUrl = canReadURLFromEnv() ? process.env['VUE_APP_BASE_URL'] : defaultUrl
+  return baseUrl + settings.apiVersion + '/'
```

Comment:
maybe I worked too much today... or the other day but there is no settings object?
should be:
```
return `${baseUrl}${API_VERSION}/`
```
or
```
return baseUrl + API_VERSION + '/'
```

Reply:
That's right, settings was our first attempt before you dropped the version number simply in the header ;-)

Reply:
I remember